src/objectTranslations/npe03__Recurring_Donation__c-ja.objectTranslation

```diff
@@ -160,58 +169,71 @@
     </fields>
     <fields>
-        <help><!-- The form of payment for this Recurring Donation. NPSP automatically copies the selected value to each related Payment record. --></help>
-        <label><!-- Payment Method --></label>
+        <help>来年または翌会計年度の継続支援の予測総額 (参照のみ)。</help>
+        <label>来年の金額</label>
+        <name>NextYearValue__c</name>
+    </fields>
+    <fields>
+        <help>この継続支援の支払のフォーム。NPSP は選択した値を関連する各支払レコードに自動的にコピーします。</help>
+        <label>支払方法</label>
         <name>PaymentMethod__c</name>
         <picklistValues>
             <masterLabel>ACH/EFT</masterLabel>
-            <translation><!-- ACH/EFT --></translation>
+            <translation>ACH/EFT</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Check</masterLabel>
-            <translation><!-- Check --></translation>
+            <translation>小切手</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Credit Card</masterLabel>
-            <translation><!-- Credit Card --></translation>
+            <translation>クレジットカード</translation>
         </picklistValues>
     </fields>
     <fields>
-        <help><!-- Default value is Open, meaning a recurring donation with no end date. --></help>
-        <label><!-- Recurring Type --></label>
+        <help>デフォルト値は [無期限] です。これは、終了日のない継続支援を意味します。</help>
+        <label>継続種別</label>
         <name>RecurringType__c</name>
+        <picklistValues>
+            <masterLabel>Fixed</masterLabel>
+            <translation>固定</translation>
+        </picklistValues>
         <picklistValues>
             <masterLabel>Open</masterLabel>
-            <translation><!-- Open --></translation>
+            <translation>Open</translation>
         </picklistValues>
     </fields>
     <fields>
-        <help><!-- The date of the first installment Opportunity for this Recurring Donation. Changes made to this field after the schedule has started will not impact future installment Opportunities or the schedule. The default is the current date. --></help>
-        <label><!-- Start Date --></label>
+        <help>この継続支援の新規および更新されたスケジュール情報 (金額、日付など) が有効になる日付。デフォルトは現在の日付です。</help>
+        <label>発効日</label>
         <name>StartDate__c</name>
     </fields>
     <fields>
-        <help><!-- Indicates if this Recurring Donation is actively in use, temporarily suspended, or closed. --></help>
-        <label><!-- Status --></label>
+        <help>継続支援が有効に使用されているのか、一時的に停止しているのか、完了しているのかを示します。</help>
+        <label>状況</label>
         <name>Status__c</name>
         <picklistValues>
             <masterLabel>Active</masterLabel>
-            <translation><!-- Active --></translation>
+            <translation>有効</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Closed</masterLabel>
-            <translation><!-- Closed --></translation>
+            <translation>Closed</translation>
```

Comment:
@GloryChoe This is also missing a translation

Reply:
@force2b I'll get this fixed asap. Can we deliver the translations with the LQA fixes as LQA should kick-off tomorrow (Tue) and return hopefully within a week or so, or do you need this sooner? Trying to minimize the overhead work and # of deliveries we ask from the vendor team. Thanks!

Reply:
I think the LQA part is up to @mpusto. For me though, I wouldn't want to release the translation in the package without these missing pick list translations.

Reply:
I'd prefer to have all the fixes in as few deliveries as possible. So yes, @GloryChoe I think delivering them together is fine.

Reply:
Thanks @mpusto 
Hi @force2b I just pulled the latest from the feature branch and we're seeing a bunch of new labels, should I get these translated for the April release? Some screenshots on the diff below.

![image](https://user-images.githubusercontent.com/44650884/77454223-78b72c80-6db5-11ea-9168-ed566457be21.png)

![image](https://user-images.githubusercontent.com/44650884/77454253-840a5800-6db5-11ea-837a-6c83171f71c3.png)

Reply:
@GloryChoe These are new labels we added after our label freeze. Do not worry about including these as they were not part of the original quote.